<!-- ar_track.launch -->
<launch>
    <!-- 设置音频文件路径参数 -->
    <param name="audio_path" value="/home/lab/.config/lejuconfig/music" type="string"/>

    <!-- 启动 播放音乐服务 -->
    <node pkg="kuavo_audio_player" type="loundspeaker.py" name="play_music_node" output="screen" respawn="true" respawn_delay="5"/>
    <param name="music_path" value="/home/lab/.config/lejuconfig/music" />
</launch>